src/components/grid-aware/TwoParagraphBlock/TwoParagraphBlock.module.css

```diff
@@ -0,0 +1,159 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background: var(--color-gray-300);
+  grid-column: 1 / span 2;
+  grid-row: 1;
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  grid-row: 1;
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+  }
+
+  .bleedBackground {
+    grid-column: 1 / span 3;
+  }
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  grid-template-rows: auto 590px;
+  padding-top: 140px;
+  row-gap: 84px;
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+    padding: 50px 0 80px;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaTitle {
+  grid-column: 2 / span 4;
+  grid-row: 1;
+}
+
+.gridAreaParagraph1 {
+  grid-column: 6 / span 6;
+  grid-row: 1;
+}
+
+.gridAreaParagraph2 {
+  grid-column: 2 / span 5;
+  grid-row: 2;
+}
+
+.gridAreaImage {
+  grid-column: 8 / span 4;
+  grid-row: 2;
+}
+
+@media (--tablet-and-down) {
+  .gridAreaImage {
+    text-align: right;
+  }
+}
+
+
+/* Component wrappers */
+
+.paragraph1Wrapper {
+  font: var(--font-body-large);
+  margin-top: 0;
+}
+
+.paragraph2Wrapper {
+  font: var(--font-body-small);
+  margin-top: 75px;
```

Comment:
Is this still desirable? I know that the component looks slightly different between the Design System mockup and the mockup of the Volunteer page, but I don't think this is needed if we have the `row-gap` defined on the main grid.

Edit: Ah, I see, in mobile, you need it in order to make sure the image doesn't overlap the paragraph. Do you think you could take this off `.paragraph2Wrapper`, and instead add it to the mobile media query version of `.image1Wrapper` as a `margin-bottom`? I think that would make it more obvious why this margin is here.

Also, I think you could reset the `top:` property to `0` on `.image1Wrapper` in mobile, since it makes the margins more complicated to reason about, and instead, you could just use normal margins. That will make it so that you don't have to do math to account for the relative offsetting, like you did with the `margin-top: 5px` in that element.

Reply:
If I add a `margin-bottom` property to `.image1Wrapper` in mobile it doesn't create a space between the image and the second paragraph block but if I add a `margin-top` property to `.paragraph2Wrapper` in mobile, it will create the spacing needed. Is this due to collapsing margins and what would be the best solution?

Reply:
Thanks for going over this with me! 👻